cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,108 @@
+/// <reference types='cypress' />
+
+import { generateWorker } from '../support/generateWorker'
+
 describe('Web Tables page', () => {
-  it('', () => {
-    
+  const { firstName, lastName, email, age, salary, department } = generateWorker();
+  
+  beforeEach (() => {
+    cy.visit('/');
+  });
+
+  it('should contain pagination', () => {
+    cy.get('.-pageJump')
+      .should('exist');
+
+    cy.get('.-previous > .-btn')
+      .should('exist')
+      .should('contain.text', 'Previous');
+
+    cy.get('.-next > .-btn')
+      .should('exist')
+      .should('contain.text', 'Next');
+  });
+
+  it('should contain rows count selection', () => {
+    const randomRowIndex = Math.floor(Math.random() * 6);
+    const rows = ['5 rows', '10 rows', '20 rows', '25 rows', '50 rows', '100 rows']
+    const row = rows[randomRowIndex];
+
+    cy.get('select')
+      .select(row);
+  });
+
+  it('should add a new worker', () => {
+      cy.get('#addNewRecordButton')
+      .should('contain.text', 'Add')
+      .click()
+
+    cy.findByPlaceholder('First Name')
+      .type(firstName);
+    cy.findByPlaceholder('Last Name')
+      .type(lastName);
+    cy.findByPlaceholder('name@example.com')
+      .type(email);
+    cy.findByPlaceholder('Age')
+      .type(age);
+    cy.findByPlaceholder('Salary')
+      .type(salary);
+    cy.findByPlaceholder('Department')
+      .type(department);
+    cy.get('#submit')
+      .click();
```

Comment:
You did function for this purpose, let's use it

Reply:
Misha mentioned in his video that the first time we should test "manually", so we can observe which field doesn't work correctly if the tests were to fall. I think he has a point with this approach so I won't change these tests.

Reply:
But you don't have tests here, you don't check the added user, you just add it, but don't check

Reply:
Why should I? This test is for adding a new worker, not for asserting that they were added.

Reply:
Why should I? This test is for adding a new worker, not for asserting that they were added.

Reply:
Ok, I tried to explain my point of view, if you don't agree with me then you can manually change the status in the spreadsheet. I just think that the whole point of testing is to test because you can type all this data, but this data might not be added for some reason and this is important to test. Just imagine what did you test? How can your code be helpful for the product, developers, and so on?